openedx/core/djangoapps/catalog/utils.py

```diff
@@ -151,16 +173,108 @@ def get_program_types():
         return []
 
 
-def get_programs_with_type_logo():
+def _get_program_instructors(program):
```

Comment:
This function is unnecessary. Fully serialized representations of programs returned by the detail endpoint include an array keyed as `staff` listing all associated instructors.